airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresDestination.java

```diff
@@ -131,33 +133,42 @@ public AirbyteConnectionStatus check(JsonNode config) {
     // connect to db.
     final Database database = getDatabase(config);
     Map<String, WriteConfig> writeBuffers = new HashMap<>();
-
+    Set<String> schemaSet = new HashSet<>();
```

Comment:
```suggestion
    final Set<String> schemaSet = new HashSet<>();
```

Reply:
How come the Map right above is not final either?